### ISSUE TITLE
normalization

### ISSUE THREAD

**Issue by nationalflag** [kenshohara/3D-ResNets-PyTorch#12]
Are the inputs of all pretrained models normalized by mean? I find you add the mean of kinetics, while there is only the mean of of activitynet several weeks ago, so could tell us which mean is used to train the pretrained models? When I finetune the pretrained models to my custom dataset, need I compute the mean of the custom dataset?

**Comment by kenshohara** [comment]
In the training of the pretrained models, I used the mean of activitynet.
I'm sorry for the confusing setting.
If you finetune our pretrained models, you should use the same means i.e. the means of activitynet.

**Comment by isht8** [comment]
For training on kinetics, you used the mean of activitynet and the std of kinetics. Correct?
